### PR TITLE
Libretro: add buildbot targets and make fixes

### PR DESCRIPTION
**Pull request by SimpleTease**

### CODE REVIEW

**Comment by SimpleTease** [comment]
Making more updates from other libretro repos. WIP status.

**Comment by SimpleTease** [comment]
Ready. I'll add more targets later if I can test building.

**Comment by SimpleTease** [comment]
Not ready. Have to make another edit.

**Comment by SimpleTease** [comment]
Okay.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,74 +1,148 @@
-.core-defs:
+# DESCRIPTION: GitLab CI/CD for libRetro (NOT FOR GitLab-proper)
+
+##############################################################################
+################################# BOILERPLATE ################################
+##############################################################################
+
+# Core definitions.core-defs:
```
Comment (m4xw) [inline .gitlab-ci.yml:7]: @SimpleTease You forgot a \n

Comment (SimpleTease) [reply .gitlab-ci.yml:7]: Got it! Thanks!

**Pull request merged**